beets/ui/__init__.py

```diff
@@ -1159,10 +1159,13 @@ def _open_library(config):
         )
         lib.get_item(0)  # Test database connection.
     except (sqlite3.OperationalError, sqlite3.DatabaseError):
+        message = ""
         log.debug(u'{}', traceback.format_exc())
-        raise UserError(u"database file {0} could not be opened".format(
+        if e.args[0] == "unable to open database file":
+            message = "It might be a permissions problem."
+        raise UserError(u"database file {0} could not be opened.%s".format(
             util.displayable_path(dbpath)
-        ))
+        )%message)
```

Comment:
Hello,
I add this in main but it gives me the message from the first `UserError` excpetion that raised before. 
```
    except sqlite3.OperationalError as e:
        if e.args[0] == "unable to open database file":
            raise UserError(u"unable to open database file. It might be a permissions problem")
```
Is there a way to do it like `db_query.InvalidQueryError` in order to check the `mutate` function in db.py?